src/appengine/libs/issue_management/issue_filer.py

```diff
@@ -240,6 +240,16 @@ def _get_from_metadata(testcase, name):
       remove_empty=True)
 
 
+def _get_additional_values(testcase, name):
+  """Get additional values from the job or fuzzer environment."""
+  values = data_handler.get_additional_values_for_variable(
+      name, testcase.job_type, testcase.fuzzer_name)
+
+  # TODO(ochang): Find a less hacky way of concatenating multiple values.
+  return values + data_handler.get_additional_values_for_variable(
+      name + '_1', testcase.job_type, testcase.fuzzer_name)
```

Comment:
Should we use ADDITIONAL_* , atleast that gets rid of this TODO, and matches slightly the style of like ADDITIONAL_ASAN_OPTIONS.

Reply:
That would be a more messy migration, since we need to update our jobs to use those vars (and handle any race conditions). 

Or are you saying to have both `AUTOMATIC_*` and `ADDITIONAL_*` ? That seems like it can be confusing too :/

Reply:
yes both AUTOMATIC_* and ADDITIONAL_*. Current version is fine too. it will always remain as a hack specific to this usecase, so unsure if todo is needed ? Can be rephrased as an global override use for a specific part of lab/platform/etc